### PR TITLE
Add benchmark for strings/fixed_point convert APIs

### PR DESCRIPTION
**Pull request by davidwendt**
Reference #5698
This PR adds benchmark tests for `cudf::strings::to_fixed_point` and `cudf::strings::from_fixed_point`.
The benchmarks include measurements for both `decimal32` and `decimal64` types.

### CODE REVIEW

File: cpp/benchmarks/string/convert_fixed_point_benchmark.cpp [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+
+  for (auto _ : state) {
+    cuda_event_timer raii(state, true);
+    volatile auto results = cudf::strings::to_fixed_point(strings_view, dtype);
+  }
+
+  using rep_type = typename cudf::device_storage_type_t<fixed_point_type>;
+  // bytes_processed = bytes_input + bytes_output
+  state.SetBytesProcessed(state.iterations() *
+                          (strings_view.chars_size() + rows * sizeof(rep_type)));
```
Comment (codereport) [inline cpp/benchmarks/string/convert_fixed_point_benchmark.cpp:59]: I think you can use `cudf::size_of` here.
```suggestion
  // bytes_processed = bytes_input + bytes_output
  state.SetBytesProcessed(state.iterations() *
                          (strings_view.chars_size() + rows * cudf::size_of(fixed_point_type)));
```

File: cpp/benchmarks/string/convert_fixed_point_benchmark.cpp [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+  for (auto _ : state) {
+    cuda_event_timer raii(state, true);
+    results = cudf::strings::from_fixed_point(fp_col->view());
+  }
+
+  using rep_type = typename cudf::device_storage_type_t<fixed_point_type>;
+  // bytes_processed = bytes_input + bytes_output
+  state.SetBytesProcessed(
+    state.iterations() *
+    (cudf::strings_column_view(results->view()).chars_size() + rows * sizeof(rep_type)));
```
Comment (codereport) [inline cpp/benchmarks/string/convert_fixed_point_benchmark.cpp:85]: Same thing as above.

**Review by codereport — APPROVED** [review]
lgtm, other than the two comments (maybe it isn't possible)

**Review by harrism — APPROVED** [review]

**Comment by harrism** [comment]
@gpucibot merge

**Pull request merged**